README.md

```diff
@@ -29,4 +42,35 @@ sudo apt update
 sudo apt install lubuntu-desktop
 ```
 
-The system will need to be re-started with the Desktop environment in order to run the Workshop. 
\ No newline at end of file
+The system will need to be re-started with the Desktop environment in order to run the Workshop. 
+
+## Installation instructions on macOS:
+
+Tested on Mojave (10.14) using Python 3.7 installed directly from Python.org. Python installed with `homebrew` should also work, but there is a [known issue](https://github.com/matplotlib/matplotlib/issues/9637#issuecomment-515081488) when scrolling on a Matplotlib figure embedded in Tkinter that causes it to crash. Anaconda Python should also work - not tested, thought.
+
+```bash
+git clone https://github.com/ImperialCollegeLondon/python-guis.git
+cd python-guis
+python3 -mvenv venv
+. venv/bin/activate
+pip install -U setuptools wheel pip
+pip install -e .
+garden install matplotlib --kivy
+```
+
+## Installation instructions on Windows 10:
+
+The following installation instructions have been tested in Windows 10 using Git Bash (might require some extra configuration) and PowerShell (you might need to disable temporarily the security directives in order to activate the virtual environment) with Anaconda Python 3.7.
+
+```bash
+git clone https://github.com/ImperialCollegeLondon/python-guis.git
+cd python-guis
+python -mvenv venv
+. venv/Scripts/activate
+pip install -U setuptools wheel pip
```

Comment:
This gave me a slightly strange error on permissions of some temporary file but seemed to work.

Reply:
That's funny, considering that the user should have full access to the `venv` folder. Well, let's leave it, for now.